services/housefy/src/main/java/com/dech/housefy/config/SecurityConfig.java

```diff
@@ -1,24 +1,26 @@
 package com.dech.housefy.config;
 
 import com.dech.housefy.security.JwtAuthenticationFilter;
+import com.dech.housefy.security.AccessFilter;
+
 import lombok.RequiredArgsConstructor;
-import org.apache.tomcat.util.net.openssl.ciphers.Authentication;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.security.authentication.AuthenticationProvider;
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
-import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
 import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
 import org.springframework.security.config.http.SessionCreationPolicy;
 import org.springframework.security.web.SecurityFilterChain;
 import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
 
 @Configuration
-@EnableWebSecurity
+@EnableMethodSecurity
```

Comment:
what is the difference between EnableWebSecurity and EnableMethodSecurity, and make sure that everything works correctly